tendermint/tendermint.github.io
# Add benefits and about to career page

## Description
Beef up careers page. 

**On individual job listing page:**
* Added 'About us' section
* Added 'Benefits' section

**On job listings index page:**
* Removed 'meta' tags from display

## Steps to test
* Check out branch
* Run `npm run dev` and go to http://localhost:8800/careers/
* It should look like this:
![image](https://cloud.githubusercontent.com/assets/484191/21403404/7cb4d294-c771-11e6-9f78-599e8256cbb6.png)

* Click on an individual job listing
* It should look like this:
![image](https://cloud.githubusercontent.com/assets/484191/21403471/bd776d8c-c771-11e6-8890-ef5114c31e64.png)
![image](https://cloud.githubusercontent.com/assets/484191/21404423/fa8b9230-c775-11e6-9a5c-e6e8ebdaeea4.png)
